src/transformers/tokenization_utils_base.py

```diff
@@ -1593,8 +1593,13 @@ def __init__(self, **kwargs):
 
         self.model_input_names = kwargs.pop("model_input_names", self.model_input_names)
 
-        # By default, cleaning tokenization spaces for both fast and slow tokenizers
-        self.clean_up_tokenization_spaces = kwargs.pop("clean_up_tokenization_spaces", True)
+        # By default, do not clean tokenization spaces for both fast and slow tokenizers
+        if "clean_up_tokenization_spaces" not in kwargs:
+            warnings.warn(
+                "The `clean_up_tokenization_spaces` argument will be deprecated and set to False by default.",
+                FutureWarning,
```

Comment:
```suggestion
            warnings.warn(
                "`clean_up_tokenization_spaces` was not set. It will be set to `True` by default. This behavior will be depracted in transformers v4.45, and will be then set to `False` by default. For more details check this issue:",
                FutureWarning,
```
and add the link to the issue.